server/internal/usecase/interactor/integration_test.go

```diff
@@ -488,7 +513,8 @@ func TestIntegration_UpdateWebhook(t *testing.T) {
 }
 
 func TestIntegration_DeleteWebhook(t *testing.T) {
-	now, op, uri, _, iId1, iId2, i1, i2 := testSuite()
+	ts := testSuite()
+	now, op, uri, iId1, iId2, i1, i2 := ts.Now, ts.Op, ts.Uri, ts.IId1, ts.IId2, ts.I1, ts.I2
```

Comment:
```suggestion
```

Directly use them in test functions because there are too many variables: e.g. `ts.Now`, `ts.Op`